tests/unit/features/RandomFourierDotFeatures_unittest.cc

```diff
@@ -176,3 +176,30 @@ TEST(RandomFourierDotFeatures, add_to_dense_test)
 
 }
 
+TEST(RandomFourierDotFeatures, random_coeff_dim)
+{
+	int32_t num_dims = 50;
+	int32_t vecs = 5;
```

Comment:
could you use consistent variable names?

Reply:
I don't quite get what you mean, num_dims(n) and vecs(m) is used throughout the unit test file. I was just following the convention.

Reply:
the point of touching this code is that it is quite badly written, and we are trying to improve things whenever we write something new or touch existing stuff. consistent variables names simply means that you use the same pattern everywhere: if you prepend "num" to denote a number, then you do don't omit this in the next variable.